bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/presentation-download-dropdown/component.jsx

```diff
@@ -91,28 +91,14 @@ class PresentationDownloadDropdown extends PureComponent {
     };
 
     if (!isDownloadable) {
-      this.menuItems.push(
-        {
-          key: this.actionsKey[0],
-          dataTest: 'enableOriginalPresentationDownload',
-          label: intl.formatMessage(intlMessages.enableOriginalPresentationDownload),
-          onClick: () => toggleDownloadOriginalPresentation(true),
-        },
-      );
-    } else {
-      this.menuItems.push(
-        {
-          key: this.actionsKey[0],
-          dataTest: 'disableOriginalPresentationDownload',
-          label: intl.formatMessage(intlMessages.disableOriginalPresentationDownload),
-          onClick: () => toggleDownloadOriginalPresentation(false),
-        },
-      );
-    }
-
-    if (hasAnnotations) {
-      this.menuItems.push(
-        {
+      this.menuItems.push({
+        key: this.actionsKey[0],
+        dataTest: 'enableOriginalPresentationDownload',
+        label: intl.formatMessage(intlMessages.enableOriginalPresentationDownload),
+        onClick: () => toggleDownloadOriginalPresentation(true),
+      });
+      if (hasAnnotations) {
+        this.menuItems.push({
```

Comment:
Consider removing the annotated download button from both if's (`if (!isDownloadable) {` and `if (hasAnnotations) {`) as it works independently of the downloadable status of the presentation and annotations count now.